openshift-knative-operator/cmd/operator/kodata/knative-eventing/0.19.2/4-mt-channel-broker.yaml

```diff
@@ -552,3 +552,62 @@ spec:
               containerPort: 8008
 
 ---
+# Copyright 2020 The Knative Authors
```

Comment:
hrm...

this should not be added, since we have a patch to remove it:
https://github.com/openshift-knative/serverless-operator/blob/master/openshift-knative-operator/hack/update-manifests.sh#L53

Reply:
outdated commit